{"name": "test_invalid_auth", "status": "passed", "start": 1743879913740, "stop": 1743879913871, "uuid": "57487ecc-50b5-4c80-a1d0-1951c334aac4", "historyId": "10bb5b361ef41c939a9774d69a07ab28", "testCaseId": "10bb5b361ef41c939a9774d69a07ab28", "fullName": "Practice.Practice_4.2.4.2.3.test_cases.TestCases#test_invalid_auth", "labels": [{"name": "parentSuite", "value": "Practice.Practice_4.2.4.2.3"}, {"name": "suite", "value": "test_cases"}, {"name": "subSuite", "value": "TestCases"}, {"name": "host", "value": "192.168.1.4"}, {"name": "thread", "value": "18208-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Practice.Practice_4.2.4.2.3.test_cases"}]}